proposals/propertyDependencies.md

```diff
@@ -0,0 +1,264 @@
+# JSON Schema Proposal: Add `propertyDependencies` Keyword
+
+## Abstract
+
+The `propertyDependencies` keyword is a more friendly way to select between two
+or more schemas to validate an instance against than is currently supported by
+JSON Schema.
+
+## Status
+
+**Current Status**: PROPOSAL
+
+TODO: We should have a short standard blurb outlining the stages involved in a
+feature making its way to stable status.
+
+TODO: Link to a document that describes the proposal => stable process in
+detail.
```

Comment:
Similar here.  I think just a link to the statuses (we should decide on stages/statuses at some point) document would suffice.

Reply:
I think it depends on how public-facing we expect this document to be. If the audience is mainly internal, then a link would be sufficient. If it's intended to be more public, I think it would be beneficial to have a short canned blurb about what the status means so causal readers can get the gist of what the status means without having to follow the link to a detailed document about our process.

Reply:
> I think it depends on how public-facing we expect this document to be.

Disagree here. We should do what the simplest is for us in this case. My feeling is that these documents are going to be pretty meaningless without a reasoable understanding of the process and where such a document currently sits within that.